examples/process-user-tasks-subsystem/pom.xml

```diff
@@ -113,6 +113,10 @@
           <groupId>org.kie</groupId>
           <artifactId>kie-addons-quarkus-persistence-jdbc</artifactId>
         </dependency>
+        <dependency>
```

Comment:
@ljmotta I saw this profile didn't had the usertasks storage, I thought on adding it.